runtime/src/main/java/org/corfudb/runtime/LogReplicationLogicalGroupClient.java

```diff
@@ -237,8 +243,55 @@ public void addDestination(String logicalGroup, List<String> remoteDestinations)
         }
     }
 
+    /**
+     * Sets a list of destinations for a group to the LogReplicationModelMetadataTable for monitoring.
+     * <p>
+     * A new logical group will be created if one does not already exist.
+     * Setting destinations for a logical group will overwrite existing destinations.
+     * Setting destinations to an empty list not, i.e. "clearing", is not permitted.
+     *
+     * @param logicalGroup The group of tables to be replicated, represented as a string.
+     * @param remoteDestinations The clusters which the specified group will be replicated to.
+     */
+    public void setDestinations(String logicalGroup, List<String> remoteDestinations) {
```

Comment:
What is the difference between this and public void addDestination(String logicalGroup, List<String> remoteDestinations)?... Were they actually asking for a setDestinations, where destinations are initially added so any call to addDestination validates against this list?

Reply:
If we were to rely on setDestinations() for removals as well, the diffs would have to be calculated on the client side which they wanted to avoid.

I think the way they phrased it for adding destinations without checking existing destinations is already taken care of in addDestination(), but if for example there was a naming change for destinations (a, b, c) to (A, B, C) this would require calls to get, remove, and add when they could just do a setDestinations() if they have them already.